src/libraries/System.Memory/tests/Span/Contains.T.cs

```diff
@@ -193,5 +195,114 @@ public static void ContainsNull_String(string[] spanInput, bool expected)
             Span<string> theStrings = spanInput;
             Assert.Equal(expected, theStrings.Contains(null));
         }
+
```

Comment:
@danmoseley I've added some tests here since there were not many tests for `Span<T>.Contains`. I believe the existing tests for `Array.IndexOf` ([link](https://github.com/dotnet/runtime/blob/7144c2f806788667a6810e87bbdb8748d0593968/src/libraries/System.Runtime/tests/System/ArrayTests.cs#L2444)) already have enough coverage. Let me know if I can add/modify anything.